core/domain/exp_services.py

```diff
@@ -378,7 +378,12 @@ def apply_change_list(exploration_id, change_list):
                 elif (
                         change.property_name ==
                         exp_domain.STATE_PROPERTY_INTERACTION_SOLUTION):
-                    state.update_interaction_solution(change.new_value)
+                    solution = None
+                    if change.new_value:
+                        solution = state_domain.Solution.from_dict(
+                            state.interaction.id, change.new_value
```

Comment:
I was expecting the params to be a `dict`.

Reply:
I cross-checked the from_dict method for the Solution object and I confirm it takes two params.
![image](https://user-images.githubusercontent.com/16653571/74780933-94b34400-52c6-11ea-8722-5a215c0d518e.png)

[Marking this comment as resolved]